complaint_search/tests/expected_results/states_agg__valid.json

```diff
@@ -0,0 +1,101 @@
+{
+  "_source": [
+            "company",
+            "company_public_response",
+            "company_response",
+            "complaint_id",
+            "complaint_what_happened",
+            "consumer_consent_provided",
+            "consumer_disputed",
+            "date_received",
+            "date_sent_to_company",
+            "has_narrative",
+            "issue",
+            "product",
+            "state",
+            "submitted_via",
+            "sub_issue",
+            "sub_product",
+            "tags",
+            "timely",
+            "zip_code"
+        ],
+  "aggs": {
+    "issue": {
+      "aggs": {
+        "issue": {
+          "terms": {
+            "field": "issue.raw",
+            "size": 5
+          }
+        }
+      },
+      "filter": {
+        "bool": {
+          "filter": [],
+          "must": [],
+          "should": []
+        }
+      }
+    },
+    "product": {
+      "aggs": {
+        "product": {
+          "terms": {
+            "field": "product.raw",
+            "size": 5
+          }
+        }
+      },
+      "filter": {
+        "bool": {
+          "filter": [],
+          "must": [],
+          "should": []
+        }
+      }
+    },
+    "state": {
+      "aggs": {
+        "state": {
+          "aggs": {
+            "issue": {
+              "terms": {
+                "field": "issue.raw",
+                "size": 1
+              }
+            },
+            "product": {
+              "terms": {
+                "field": "product.raw",
+                "size": 1
+              }
+            }
+          },
+          "terms": {
+            "field": "state",
+            "size": 0
+          }
+        }
+      },
+      "filter": {
+        "bool": {
+          "filter": [],
+          "must": [],
+          "should": []
+        }
+      }
+    }
+  },
+  "from": 0,
+  "query": {
+    "query_string": {
+      "default_operator": "AND",
+      "fields": [
+        "complaint_what_happened"
+      ],
+      "query": "*"
+    }
+  },
+  "size": 0
+}
```

Comment:
This linting error crops up in a couple of places in the PR. Do we mind no newline?

Reply:
where do you get that?  We obviously don't do JS linting on the Python repo.

Reply:
Github's showing that in the Pull Requests / Files tab. It IS interesting that they'd be so opinionated. I wasn't really expecting it!
![Screen Shot 2020-06-29 at 4 53 34 PM](https://user-images.githubusercontent.com/1028659/86054968-18293280-ba29-11ea-9d85-b426797e1406.png)